kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -63,17 +66,21 @@ public S3SinkTask() {
     // no-arg constructor required by Connect framework.
     assignment = new HashSet<>();
     topicPartitionWriters = new HashMap<>();
+    time = new SystemTime();
```

Comment:
I can't use the Mock from `org.apache`. That's why I've been forced to use the `io.confluent` in the code and the tests.